charts/ceph-csi-cephfs/values.yaml

```diff
@@ -41,6 +41,10 @@ logLevel: 5
 # sidecarLogLevel is the variable for Kubernetes sidecar container's log level
 sidecarLogLevel: 1
 
+
+CSIDriver:
+  fsGroupPolicy: "File"
```

Comment:
can you list the values that are supported, or link to the kubernetes documentation with them?

Reply:
done